src/main/java/taxi/controller/car/GetAllCarsController.java

```diff
@@ -3,13 +3,15 @@
 import java.io.IOException;
 import java.util.List;
 import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import taxi.lib.Injector;
 import taxi.model.Car;
 import taxi.service.CarService;
 
+@WebServlet(urlPatterns = {"/cars/all", "/cars", "/cars/"})
```

Comment:
```suggestion
@WebServlet(urlPatterns = {"/cars", "/cars/"})
```

<img width="536" alt="image" src="https://user-images.githubusercontent.com/68428976/180935162-5a70c8ee-b69c-4aa2-9cd8-39a3bf456613.png">